es_ES/faq.md

```diff
@@ -0,0 +1,625 @@
+[diagrama de arquitectura](https://docs.google.com/presentation/d/1cw7A4HbvM_Abv320rVgPVGiUP2msVs7tfGbkgdrTy0I/edit#slide=id.gbb3c3233b_0_162) para una mejor idea de los componentes principales.
+
+### Cómo ejecutar Flutter mi código en Android? {#run-android}
+
+Los códigos C y C++ del motor están compilados con el NDK de Android. El código Dart
+(tanto los SDK como los tuyos) están anticipados (AOT) compilados en una biblioteca ARM nativa. Esa biblioteca está incluida en un proyecto de Android "runner", y el conjunto está integrado en un APK. Cuando se inicia, la aplicación carga la biblioteca Flutter.
+Cualquier representación, entrada o manejo de eventos, etc., se delegan en el compilado
+Flutter y código de la aplicación. Esto es similar a la forma en que funcionan muchos motores de juegos.
+
+Las construcciones de modo de depuración usan una máquina virtual (VM) para ejecutar el código de Dart (de ahí la "depuración"
+anuncio que muestran para recordar a las personas que son un poco más lentos) con el fin de
+habilitar el stateful hot reload.
+
+### Cómo funciona Flutter para ejecutar mi código en iOS? {#run-ios}
+
+Los códigos C y C++ del motor están compilados con LLVM. El código Dart (tanto
+Los SDK y los suyos) están anticipados (AOT) compilados en una biblioteca ARM nativa.
+Esa biblioteca está incluida en un proyecto de iOS "runner", y todo está construido
+en un `.ipa`. Cuando se inicia, la aplicación carga la biblioteca Flutter. Cualquier renderizado, manejo de entradas o eventos, y así sucesivamente, se delegan al compilado de Flutter y código de la aplicación. Esto es similar a la forma en que funcionan muchos motores de juegos.
+
+Las construcciones de modo de depuración usan una máquina virtual (VM) para ejecutar el código de Dart (de ahí el banner de "depuración" que muestran para recordar a las personas que son un poco más lentos) con el fin de
+habilitar stateful hot reload.
+
+### Flutter usa los widgets OEM de mi sistema?
+
+No. En cambio, Flutter proporciona un conjunto de widgets
+(incluidos los widgets de Material Design y Cupertino (estilo iOS))
+administrado y renderizado por el framework y el motor de Flutter.
+Puedes navegar por un
+[catálogo de widgets de Flutter](/widgets/).
+
+Esperamos que el resultado final sean aplicaciones de mayor calidad. Si reutilizamos
+los widgets OEM, la calidad y el rendimiento de las aplicaciones de Flutter serán
+limitado por la calidad de esos widgets.
+
+En Android, por ejemplo, hay un conjunto de gestos rígidos y reglas fijas
+para desambiguarlos. En Flutter, puede escribir su
+propio gestor de reconocimiento que es un participante de primera clase en el
+[gesture system](/gestures/). Por otra parte, dos widgets
+escrito por diferentes personas puede coordinar para desambiguar los gestos.
+
+Las tendencias de diseño de aplicaciones modernas apuntan a diseñadores y usuarios que desean UI más ricas en movimiento y diseños propios de la marca.

```

Comment:
Modern app design trends point towards designers and users wanting more motion-rich UIs and brand-first designs

Las tendencias de diseño en las aplicaciones modernas apuntan hacia que diseñadores y usuarios esperan UIs más rica en movimientos y diseños propios de la marca.